src/css/blocks/card.css

```diff
@@ -0,0 +1,92 @@
+.card {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.card__picture {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__title {
+  width: 100%;
+  height: 36px;
+  margin: 0 16px;
+  max-width: 166px;
```

Comment:
wow.. why do we need max-width for the title?

Avoid coping styles from Figma🥲

Reply:
fixed